02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,116 @@
-# Introducing survey data {#c02}
+# Overview of Surveys {#c02}
+
+Creating and fielding surveys to provide estimates of a population are much more complex than simply adding a few questions to an online platform. Survey researchers can spend months or even years developing the study design, questions, and other methods for a single survey to ensure high quality data is collected. While this book focuses on the analysis methods of surveys, understanding the entire survey process can provide a better understanding of what types of analyses should be conducted on the data. There are many pieces of existing literature that cover designing and implementing surveys that we recommend reviewing in detail (e.g., {cite: Dillman book, Groves book, education book?, others?}). The information provided in this chapter is a brief introduction into the survey process aiming to provide a foundation of the complexities of survey data.
+
+The survey life cycle starts with a research topic or question of interest. For example, a researcher may be interested in the impact childhood trauma has on political affiliation later in life. Before choosing to conduct a survey, researchers typically review existing data sources to determine if data are already available that can answer this question. If existing data cannot answer the nuances of the research question, a survey can be used to capture the exact data that the researcher needs.
+
+Figure \@ref(fig:svylifecycle) shows a high level view of the steps that researchers must conduct to go from a research question to final results.
+
+<!-- Note: need to adjust the size of the text in this figure. -->
+
+```{r svylifecycle, echo=FALSE, fig.cap = "Overview of the survey process"}
+library(DiagrammeR)
+mermaid("
+graph TD
+  A[Survey Concept]-->B[Study Design]
+  B-->C[Questionnaire Design]
+  B-->D[Data Collection Materials]
```

Comment:
Data Collection Materials might be too jargony for audience unless defined outside of diagram.